blocklytest/src/androidTest/java/com/google/blockly/model/BlockTestStrings.java

```diff
@@ -59,7 +59,7 @@
             + "    {"
             + "      \"type\": \"field_checkbox\","
             + "      \"name\": \"NAME\","
-            + "      \"checked\": true"
+            + "      \"checked\": TRUE"
```

Comment:
This isn't valid JSON.  true is a boolean value. "TRUE" is a string.  Just TRUE is neither.